vms/platformvm/txs/builder/builder.go

```diff
@@ -49,6 +50,7 @@ type AtomicTxBuilder interface {
 		to ids.ShortID,
 		keys []*secp256k1.PrivateKey,
 		changeAddr ids.ShortID,
+		memo []byte,
```

Comment:
we zero memo fields upon Durango activatio, but we may still want to specify them in UTs

Reply:
I think two main points here:

- We tend to have txBuilder as general as possible, allowing them to specify fields that latest fork has dropped (see start time in AddValidator/Delegator tx)
- We may reactivate memos in the future, once we price them correctly

On the other side:

- Dropping start time from AddValidator/Delegator txs builder would have been a major pain due to extensive use in UTs. That's not the case for the memo, which is never specified except in the single UT were we show it's disabled with Durango. I judged it's best to introduce them in builders (as we do in the wallet itself as well), but I think added value is definitely not a large one. I could have avoid specifiing the memo field and build "manually" the txs in that single UT. I just dislike this alternative as bulding txs with dynamic fees requires more complex code and I strived to avoid code duplication as much as possible.